README.md

```diff
@@ -16,6 +16,11 @@ Accordingly, if you update to Git 2.9 or later, you can use this alias instead o
 
     git config --global alias.up 'pull --rebase --autostash'
 
+If you wish you to make it so that this happens on every `git pull` in Git 2.9, you can also set that by doing:
```

Comment:
Could this be reworded to:

    If you'd rather this happened on every `git pull`, then you can do this if you're running Git 2.9 or later: